### DISCUSSION TITLE
Support udp2raw/stunnel application to baypass blocking in countries

### DISCUSSION THREAD

**Discussion by EsmailELBoBDev2** [angristan/openvpn-install#966] [Ideas]
I'm a user in one of countries where i can't use both actually wireguard and openvpn because my ISP is blocking both protocols 

So it would be nice idea to add support for application like udp2raw or really any other application to unblock udp protocol

heck! even if you want i could submit an PR to add it

**Comment by TinCanTech** [comment]
Submit submit submit!

**Comment by EsmailELBoBDev2** [comment]
Sure! i already got guides link just allow me some time to translate it into script and merge it 

PS: *my exams are near (24th this month so yup you need to wish me luck and allow me some time buddy!) but wish me luck i'm serious i did not study well*

**Comment by Nuruddinjr** [comment]
@EsmailELBoBDev2  Hi, what happened to this after that?

**Reply by EsmailELBoBDev2** [reply]
Well, Long story short udp2raw stopped to work for some reason and so far I'm still looking. I wrote a draft blog if you Interested https://blog.esmailelbob.xyz/d/nn3jjiw82o

Right now I'm trying to search to make wireguard play nice with v2ray but whenever I run wg as vpn it does not work and whenever I write wg with split tunnel it works fine so this is like only step i need to figure it out

**Comment by bobparkerbob888-tech** [comment]
wstunnel could work
